src/server/controllers/usersController.js

```diff
@@ -86,39 +86,39 @@ const updateUser = async(req, res) => {};
 const deleteUser = async(req, res) => {};
 
 //Action to return list user's based on the firstname
-const search = async (req, res) => {
-  const firstname = req.query.term;
-
-  const users = await User.find({
-    firstname: { $regex: firstname, $options: "i" },
-  }).then((users) => {
-    if (users) {
-      res.status(200).json(users);
-    } else {
-      return res.status(400).json({ message: "No user exists with this name" });
-    }
-  });
+const search = async(req, res) => {
+    const firstname = req.query.term;
+
+    const users = await User.find({
+        firstname: { $regex: firstname, $options: "i" },
+    }).then((users) => {
+        if (users) {
+            res.status(200).json(users);
+        } else {
+            return res.status(400).json({ message: "No user exists with this name" });
+        }
+    });
```

Comment:
i have a controller that has the same name "search" i believe, will this make any conflicts? If possible can you change this name